cypress/Dockerfile

```diff
@@ -1,17 +1,20 @@
 ARG NODE_VERSION='20.9.0'
```

Comment:
Looks like this env variable is no longer used.  Could this be integrated into line 5 to drive the node version of the Cypress image?